src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,194 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * <h>duke.Storage</h>
+ * Deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    private final File file;
+
+    /**
+     * Constructor for duke.Storage class.
+     * @param filePath The path of the file todolist.txt.
+     * @throws IOException On input error.
+     */
+    public Storage(String filePath) throws IOException {
+        this.file = new File(filePath);
+        boolean exists = file.exists();
+        if (!exists) {
+            file.createNewFile();
+        }
+    }
+
+    /**
+     * Getter to get the file.
+     * @return File The file.
+     */
```

Comment:
Probably removing the javadocs for trivial methods like getters and setters. This is just a small suggestion.